### ISSUE TITLE
Changing ExportLinkedFiles of IFCExportConfiguration doesn't have any affect

### ISSUE THREAD

**Issue by ibober** [Autodesk/revit-ifc#222]
I have links in my Revit project. If I export manually with "Export linked files as separate IFCs" checkbox enabled I got several IFC files in the output.
But if I use `IFCExportConfiguration` class from IFCExportUIOverride.dll to update my `IFCExportOptions` and set `ExportLinkedFiles` bool value to true I still got only one IFC file without links.

**Comment by YacineBougrinatAGMT** [comment]
@ibober were you able to resolve this?

**Comment by ibober** [comment]
> @ibober were you able to resolve this?

Nope. As far as I remember, I had to programme copying elements from link into the main model temporary than reverting this.

**Comment by YacineBougrinatAGMT** [comment]
Appreciate your feedback, thanks!

**Comment by jeremytammik** [comment]
I [asked](https://autodesk.slack.com/archives/C0SR6NAP8/p1687865380040519) the development team for you.

**Comment by Gopinath-Rajendran** [comment]
@jeremytammik  did you get any feedback from the development team?

**Comment by ArtemHnatko** [comment]
This issue is also reported in Jira as "REVIT-252509" and is currently in the "Open" status.
